app/src/main/java/aashrai/android/gettowork/view/activity/SettingsActivity.java

```diff
@@ -49,6 +50,8 @@
   }
 
   private void configureSearch() {
+   search.setCompoundDrawables(null, null,
```

Comment:
True but its recommended that we use the activity context while getting the theme and not the application theme